### ISSUE TITLE
tls-xb can't work properly if there are any spaces in username 

### ISSUE THREAD

**Issue by wusitee** [hey2022/tls-xb#40]
When the username or the password contains Chinese character, tls-xb (0.3.5) can't log in and fetch data from Tsinglan-Xiaobao
It works on older version.

**Comment by hey2022** [comment]
I cannot reproduce on v0.3.5 with a password containing Chinese characters, and I cannot find a way to change my username to contain Chinese characters.
Can you test whether Chinese characters in the username, or in the password, is causing the problem?

**Comment by aulimaru** [comment]
Cannot reproduce on commit [576a786](576a786152af5e8129ba34a883ca48bb2c33e103) on Linux

**Comment by hey2022** [comment]
> Cannot reproduce on commit [576a786](576a786152af5e8129ba34a883ca48bb2c33e103) on Linux

Did you try to reproduce with Chinese characters in username, password, or both?

**Comment by wusitee** [comment]
In the current git version, when username contains Chinese character it still can't log in, and when trying re-login, Username and Password will be on one line.
Like this: [Copy-Q-v-XBt-XL.png](https://postimg.cc/B82m7NSM)

**Comment by hey2022** [comment]
Since you can reproduce, can you do a git bisect and find the offending commit.

**Comment by wusitee** [comment]
how

**Comment by hey2022** [comment]
https://git-scm.com/docs/git-bisect

**Comment by wusitee** [comment]
Commit c3c2de5055695bc07922295884442955f481755f causes the issue. If the username contains Chinese character, such as `学生 student`, the `name` store in the config file will only contain the Chinese part which excludes `student`, like `学生`. 

If we let  `学生 student` and `123456` be the input:

The output on v0.3.4 should be:
```toml
name = "学生 student"
password = "2788C3EFB886D065E6F08FF19DB186DD"
timestamp = 1731238250
```

While the output on v0.3.5 or the later version of `config.toml` is:
```toml
name = "学生"
password = "D39CF2715890F528A89223039F5288E3"
timestamp = 1731238448
```

**Comment by wusitee** [comment]
Seems like it isn't the Chinese character issue. If the user name contains space, the result will only be the first part of the input.

**Comment by hey2022** [comment]
I think we both know the problem now and how to fix it.

**Comment by wusitee** [comment]
Should be fixed by #55
